modules/name_day.py

```diff
@@ -0,0 +1,13 @@
+from requests import get
+import json
+
+def todays_names(line):
```

Comment:
The function requires a paramter 'line' which is never used. The paramter is also not specified in the actuator call (line 124 of main.py), which causes a TypeError when called. If you remove the unused paramter I would be more than happy to merge the PR :)

``` suggestion
def todays_names():
```

Reply:
Yeah, right. I messed it up.

Reply:
Hotfix have been pushed.